src/components/MessageBubble.js

```diff
@@ -18,19 +19,33 @@ class MessageBubble extends Component {
     render() {
       return (
         <MessageBubbleContainer isAuthor={this.state.isAuthor} backgroundColor={this.props.labelColor} isSelectable={this.props.isSelectable}>
-          <TextInputContainer placeholder='Message' isConfirmed={false}>
+           <WrittenMessage
+           placeholder='Message'
```

Comment:
Add this and the following lines in one line in the tag. Will be easier to read.